### PR TITLE
Play Lichess ongoing games and accept challenges

### PR DESCRIPTION
**Pull request by bpycinski**
This PR extends main menu with Lichess-specific entries: show ongoing games and challenges (both incoming and outcoming, created externally). 

The ongoing games should be in the initial position (no moves played so far).

This PR has already merged PR #155 by @joaomrcarvalho, as that one contains current hot-fixes.

### CODE REVIEW

**Comment by djaramil** [comment]
@bpycinski, how can I get a build that I can try with this new function? or what's the best way to try this out?   Also, on the current code base 1.2.1, I tried to put the lichess api key in throught the web interface, and then tried the lichess option on the board and it gave me a error saying "Error in API Token".  I've given it the permissions shown in the website.

**Comment by bpycinski** [comment]
Hi @djaramil , nice to hear you are interested in this PR. 
I run this code with a little manually effort, just clone the branch somewhere into the Pi, and copy from the repository a folder `DGTCentaurMods/DGTCentaurMods/opt/DGTCentaurMods/` to a destination `/opt/DGTCentaurMods/` (there are just a few files modified). I advise to make a backup of this directory first.
After updating the files, a simple command is sufficient: `sudo systemctl restart DGTCentaurMods.service`. Or `stop` the service and run the code manually to get the logs onto the console: `cd /opt/DGTCentaurMods;  PYTHONPATH=/opt  python3 game/menu.py`

File: requirements.txt
Comment (wormstein) [inline requirements.txt:1]: These are already in  `/DGTCentaurMods/opt/DGTCentaurMods/setup/requirements.txt` and installed by pip3 during `postinst` script.

File: DGTCentaurMods/opt/DGTCentaurMods/game/menu.py [hunk 1]
```diff
@@ -230,6 +233,26 @@ def show_welcome():
...
+    client = berserk.Client(session=session)
+    # just to test if the token is a valid one
+    try:
+        who = client.account.get()
+    except berserk.exceptions.ResponseError:
+        logging.error('Lichess API error. Wrong token maybe?')
+        raise
+    return client
+
+
```
Comment (wormstein) [inline DGTCentaurMods/opt/DGTCentaurMods/game/menu.py:255]: I would put this function somewhere out of the menu.py, like inside a module containing lichess helper functions in `/board` so that we don't get lost looking where a function is defined. note: all modules in `/board` are loaded by menu.py at start.

File: DGTCentaurMods/opt/DGTCentaurMods/board/board.py [hunk 2]
```diff
@@ -51,7 +51,7 @@
 
 # Various setup
 try:
-    ser = serial.Serial("/dev/ttyS0", baudrate=1000000, timeout=0.2)
+    ser = serial.Serial("/dev/serial0", baudrate=1000000, timeout=0.2)
```
Comment (wormstein) [inline DGTCentaurMods/opt/DGTCentaurMods/board/board.py:54]: If this is not a problem with the Raspberry Pi device and is only to accommodate other possible SBC devices that were not completely tested yet, can we leave it ttyS0?

File: DGTCentaurMods/opt/DGTCentaurMods/tmp/lichessapi.py
Comment (wormstein) [inline DGTCentaurMods/opt/DGTCentaurMods/tmp/lichessapi.py:1]: I'm not sure here. Could you tell me why is this needed and why is it in `/tmp`? We store the lichess token in `//DGTCentaurMods/opt/DGTCentaurMods/config/centaur.ini` and manage it using `ConfigSystem` class that's in `/board/centaur`

File: DGTCentaurMods/opt/DGTCentaurMods/board/board.py (hunk 2 above)
Comment (bpycinski) [reply DGTCentaurMods/opt/DGTCentaurMods/board/board.py:54]: We had a discussion about that a few months ago, and this line was merged in a commit 308b782 anyway.

File: requirements.txt
Comment (bpycinski) [reply requirements.txt:1]: Right, it was again my internal file. Removed.
